CONTRIBUTING.md

```diff
@@ -24,14 +18,36 @@ The recipes do NOT cover:
 
 If you're interested in adding new chapters for any of these sections, [create an issue](https://github.com/substrate-developer-hub/recipes/issues/new) and convince us :)
 
-## Getting Involved <a name = "involve"></a>
+## Chef's Workflow <a name = "workflow"></a>
+
+https://substrate.dev/recipes/base/kitchenoverview.html
+
+TODO: break the algorithm into 2-sentence paragraphs with more specificity per paragraph
 
-1. isolate useful pattern (in [`issues`](https://github.com/substrate-developer-hub/recipes/issues))
-2. build a module/runtime example with context (in [`kitchen`](https://github.com/substrate-developer-hub/recipes/tree/master/kitchen))
+*algorithm*
+1. isolate useful pattern and make an [`issues`](https://github.com/substrate-developer-hub/recipes/issues)
+2. clone and a fork a branch; build a module/runtime example with context in the [`kitchen`](https://github.com/substrate-developer-hub/recipes/tree/master/kitchen) directory
 3. walk through logic of useful pattern in piecewise steps (in [`src/`](https://github.com/substrate-developer-hub/recipes/tree/master/src))
 4. link `src` and `kitchen` (in [`src/`](https://github.com/substrate-developer-hub/recipes/tree/master/src)) 
 
-## Local Build Instructions <a name = "instructions"></a>
+### TODO: Cooking Modules
+
+TODO:
+* preferences on style?
+* enforcement of `cargo fmt`
```

Comment:
I vote yes on cargo fmt. :+1: 

I also vote yes on some standard format for Cargo.toml. I don't have a strong preference what that format is yet.

Reply:
Cool, I just added a commit with my preference for `Cargo.toml` and an example in `adding-machine`.

Depending on your thoughts, tomorrow I'll consider clean sweeping the modules and enforcing
1. the preferred `Cargo.toml` format
2. `cargo fmt` so I can add a line to the github actions to enforce it